libs/feature/record/src/lib/data-view-map/data-view-map.component.ts

```diff
@@ -172,6 +172,15 @@ export class DataViewMapComponent implements OnInit, OnDestroy {
         type: MapContextLayerTypeEnum.WMS,
         name: link.name,
       })
+    } else if (this.linkHelper.isWmtsLink(link)) {
+      return this.mapUtils.getWmtsOptionsFromCapabilities(link).pipe(
+        map((options) => ({
+          url: link.url,
```

Comment:
I might be wrong but sounds like to me that you are using only `type` and `options` properties.
I guess the `url` is already in the `options`.
What is the `name` used for BTW in other layers ?

Reply:
Good catch, thanks, indeed `name` and `url` are already in the `options` for WMTS. The other layer types need the `name` for the layer name when requesting data/maps.